language/move-lang/src/parser/syntax.rs

```diff
@@ -212,7 +212,7 @@ where
 
 // Parse an identifier:
 //      Identifier = <IdentifierValue>
-fn parse_identifier<'input>(tokens: &mut Lexer<'input>) -> Result<Spanned<String>, Error> {
+fn parse_identifier<'input>(tokens: &mut Lexer<'input>) -> Result<Name, Error> {
```

Comment:
I didn't realize we had this "Name" type. I was hoping we could rename "ModuleAccess" to "Name" (since it is now uses for arbitrary names, including locals). Maybe "Name" could be changed to "Identifier" to avoid the conflict? Anyway, that's unrelated to this PR....

Reply:
Would be a big change, but maybe not a bad one. (`Name` is used all over for anything `Spanned<String>`). 

If you changed this you would also need to change a bunch of other things maybe? i.e. the name fits well since `StructName` is a wrapper around `Name`, i.e. `struct StructName(pub Name)`. I think it would be weird to have `struct StructName(pub Identifier)`

Reply:
Hmm. Maybe you can help me think about another alternative for ModuleAccess....